TestFabric/src/main/resources/mods.groovy

```diff
@@ -0,0 +1,124 @@
+final mdg = FabricModsDotGroovy.make {
+    schemaVersion = 1
+    id = "examplemod"
+    name = "Example Mod"
+    description = "This is an example description!"
+    version = "1.0.0"
+    provides = ["example_mod"]
+    environment = Environment.ANY
+    license = ["MIT", "CC0-1.0"]
+
+    icon {
+        it."16" = "small.png"
+        it."32" = "medium.png"
+    }
+    icon 64, "large.png"
+
+    entrypoints {
+        main "net.fabricmc.example.ExampleMod"
+        main {
+            adapter = "kotlin"
+            value = "hello.kotlin.World"
+        }
+        client "net.fabricmc.example.ExampleModClient"
+        rei = "net.fabricmc.example.ReiPlugin"
```

Comment:
Bug: setting rei twice adds rather than replaces:
```groovy
rei = "net.fabricmc.example.ReiPlugin"
rei = "net.fabricmc.new.ReiPlugin"
```
```json
"rei": [
    "net.fabricmc.example.ReiPlugin",
    "net.fabricmc.new.ReiPlugin"
],
```